modules/os2forms_webform_maps/src/Element/WebformLeafletMapField.php

```diff
@@ -104,11 +106,11 @@ public static function processWebformMapElement(&$element, FormStateInterface $f
       'gestureHandling' => FALSE,
       'reset_map' => [
         'control' => FALSE,
-        'options' => '{"position":"topleft","title":"Reset View"}',
+        'options' => '{"position":"' . $element['#position'] . '","title":"' . $reset_title . '"}',
```

Comment:
This will break, i.e. generate invalid JSON, if `$reset_title` contains `"`. Use `json_encode(['position' => $element['#position'], 'title' => $reset_title])` to compute the JSON value.